multicluster/verify_kind.sh

```diff
@@ -25,11 +25,18 @@ if ! which kind > /dev/null; then
         sudo mv kind /usr/bin
     fi
 fi
-echo ${KUBECONFIG}
+
 kind create cluster --name=antrea-leader
```

Comment:
I suggest to make `verify_kind.sh` to a general script which will verify kind only, and all other steps is for integration test, it's better to have a stand alone script. btw, you should move `verify_kind.sh` to hack folder. usually we avoid to add files in root directory.